tools/linter_lib/custom_check.py

```diff
@@ -966,6 +966,19 @@
     langs=["txt", "text", "yaml", "yml"],
     rules=whitespace_rules,
 )
+
+svg_rules = RuleList(
+    langs=["svg"],
+    rules=[
+        {
+            "pattern": r"fill=(['\"])(.*?)\1",
+            "description": """Do not specify fill for icon SVGs.
+            We should let the system using the icons decide the fill color.""",
```

Comment:
I am not too sure about the error message, suggestions welcome!

Reply:
As @andersk has pointed out elsewhere, what's actually happening is that `fill` defaults to black, in accordance with [the SVG spec](https://www.w3.org/TR/SVG2/painting.html#SpecifyingFillPaint). So the message here can be something along those lines: "System icons ignore fill values, so do not include the fill property."

Reply:
Thank you! Addressed this point!